### DISCUSSION TITLE
Stamps and Text Boxes with adjsutable transparency

### DISCUSSION THREAD

**Discussion by jaggeri** [JakubMelka/PDF4QT#52] [Ideas]
Hi Jakub,

Stamps and text boxes with adjustable transparency would be a very useful feature. I currently use this feature in Acrobat 10. I can stamp "paid" on bills, or tick with a stamp, or add a large text box (with transparency so i can read info underneath), and then combine and drag and drop thumbnails  in to one searchable "bills2023.pdf" file for the year.

**Comment by JakubMelka** [comment]
Hello,

There are several ways how to do that in my software. You can create a stamp with full color, and then set the transparency in annotation settings dialog. See word DRAFT in example image:
![image](https://github.com/JakubMelka/PDF4QT/assets/11367048/2e5238bd-7201-4eae-8634-fe90a1faeb58)

However, you can also use plugins to do that - I have signature plugin, which has many features and can create "signature" with graphic you like (choose electronic signature so it is not signed document by certificate, but just adjusted graphics).
